calcom/cal.com
# If no location is set, default to Cal Video

## What does this PR do?



This PR adds a Cal Video link to bookings if the event type location is left blank.

Fixes #5411



https://www.loom.com/share/47afc5fe2bb041f9b56005c15b3906d9

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Create an event type without a location
- Create a booking
- The success page should show web conferencing info in the confirmation email, and the confirmation email should have a cal video link

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
